src/test/java/org/opensearch/security/dlic/rest/api/SavedTenantApiTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The license header is old. Could you please use [this one](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/setting/TransportPassiveAuthSetting.java#L1-L14)?

Reply:
Changed